library/src/main/java/com/kizitonwose/calendarview/ui/DayHolder.kt

```diff
@@ -55,12 +57,12 @@ internal class DayHolder(private val config: DayConfig) {
         }
 
         if (currentDay != null) {
-            if (containerView.visibility != View.VISIBLE) {
-                containerView.visibility = View.VISIBLE
+            if (!containerView.isVisible) {
```

Comment:
@kizitonwose Went ahead and changed these too for consistency

Reply:
Makes sense!